app/assets/src/layout.njk

```diff
@@ -17,12 +17,10 @@
     {% endif %}
   {% endblock %}
   {% block header %}
-    {% set phase_banner_html %} This is a new service – your
-    <a class="govuk-link" href="{{ surveyLink }}" target="_blank" rel="noopener noreferrer">feedback</a>
-    will help us to improve it. {% endset %}
+    {% set phase_banner_html %} This is not a full service. It is not production code and you might experience problems. {% endset %}
```

Comment:
I was following the prototype here: https://grants-prototype2.herokuapp.com/new.

Since we do not have a qualtrics survey for this grant yet shall we keep it as is? WDYT?